plane/source/docs/quadplane-weathervaning.rst

```diff
@@ -76,14 +76,16 @@ note:* not QLOITER take-offs and landings
 
 .. note:: Weathervaning can be disabled or enabled by an :ref:`RC Aux Function Switch<common-auxiliary-functions>`, option "160"
 
-Using the Forward Motor
------------------------
+Using the Forward or Tilt Motors to Help Position Holding
+---------------------------------------------------------
 
 In addition to active weathervaning, the QuadPlane code supports using
 the forward motor to hold the pitch level in VTOL flight modes. To
-enable use of the forward motor for position hold you need to set the
+enable use of the forward motor(s) for position hold you need to set the
 :ref:`Q_VFWD_GAIN <Q_VFWD_GAIN>` parameter to a non-zero value.
 
+.. note:: Tailsitters do not have this option and :ref:`Q_VFWD_GAIN <Q_VFWD_GAIN>` does nothing.
```

Comment:
I suspect `Q_VFWD_GAIN` will actually do something, it will just be the wrong thing..... we should probably fix.

Reply:
so should we recommend that it be set to zero for Tailsitters?

Reply:
yes

Reply:
fixed